gbasis/parsers.py

```diff
@@ -146,18 +147,26 @@ def parse_gbs(gbs_basis_file):
             # if len(angmom_seg) == 1:
             #     coeffs_seg = coeffs_seg[:, None]
             for i, angmom in enumerate(angmom_seg):
+
+                # dummy fix for np.allclose
```

Comment:
Please make this more descriptive ("check if previous exponents are the same as these exponents")